database/migrations/2023_12_20_063600_create_authentications_table.php

```diff
@@ -11,22 +11,20 @@
      */
     public function up(): void
     {
-        Schema::create('users', function (Blueprint $table) {
+        Schema::create('authentications', function (Blueprint $table) {
             $table->id();
-            $table->string('name');
-            $table->string('email')->unique();
-            $table->timestamp('email_verified_at')->nullable();
-            $table->string('password');
-            $table->rememberToken();
+            $table->string('token')->unique();
+            $table->string('email');
+            $table->integer('status');
+            $table->timestamp('expiration_at')->default(now()->addMinutes(15));
```

Comment:
```suggestion
            $table->timestamp('expiration_at');
```
このサービスが実際にリリースされた後に有効期限の時間を1時間に延ばして欲しい依頼があった際やパスワードのリセットの認証でこのテーブルを使用する際などに有効期限が15分以外だった場合にはmigrationファイルの修正とコードの修正も必要になってきます。

なので、defaultの値はなくして、authenticateのレコードが作成されるタイミングで有効期限の時間を現在時刻から算出して保存するようにしてください。

Reply:
デフォルトの値を設定しないようにマイグレーションファイルを変更致しました。
ご確認よろしくお願い致します。
database/migrations/2023_12_20_063600_create_authentications_table.php